hare/messagevalidation.go

```diff
@@ -105,12 +105,12 @@ func (validator *syntaxContextValidator) SyntacticallyValidateMessage(m *Msg) bo
 	}
 
 	if m.InnerMsg.Values == nil {
-		validator.Warning("Syntax validation failed: Values is nil in msg: %v", m)
+		validator.Warning("Syntax validation failed: Values is nil in msg: %v", m.String())
```

Comment:
Ok, though I am wondering if we better have an err return value and than it can be printed in the most upper level for which it is relevant. It will also uncouple many other inner analyzers from the log object. What do you think?

Reply:
error should be returned only when an error occurs and not to describe a reason for some output